spec/lucky/params_spec.cr

```diff
@@ -14,6 +14,21 @@ describe Lucky::Params do
     dup_params.get(:from).should eq "form"
   end
 
+  it "works when parsing multipart params twice" do
+    request = build_multipart_request form_parts: {
+      "user" => {
+        "name" => "Paul",
+        "age"  => "28",
+      },
+    }
+
+    params = Lucky::Params.new(request)
+    dup_params = Lucky::Params.new(request)
+
+    params.nested(:user).should eq({"name" => "Paul", "age" => "28"})
+    dup_params.nested(:user).should eq({"name" => "Paul", "age" => "28"})
```

Comment:
I don't think you need to check the contents. You can probably just call `prams.nested(:users)` and `dup_params.nested(:user)` to make it clear that that is all it takes to test that multipart param parsing works

Reply:
Fair enough, I've also removed the contents checks from https://github.com/luckyframework/lucky/pull/255/files/1f0c4c289a0507b6f075db59d8cf915813d331b0#diff-8cbcd6e8b8829525283cc450f05a0166R6

Only concern is if the second attempt ever returned a blank string? If I understand things correctly, `gets_to_end` on an IO stream will initially return the contents of the IO, but a second call will continue from where it left off giving a blank string.